firmware/controllers/algo/rusefi_types.h

```diff
@@ -75,6 +75,17 @@ using efitimems64_t = int64_t;
  */
 using efitimems_t = uint32_t;
 
+// date-time struct a la ctime struct tm
```

Comment:
should this be elevated outside `controllers/algo`/`rusefi_types.h`?  what about the other stuff in here; it seems the header is used outside of this scope already, so I'm not terribly concerned; this seems like the best place for this type right now.